docs/README.md

```diff
@@ -40,7 +40,7 @@ The full documentation is available [on Read the Docs](https://dj-stripe.github.
 
 ## Installation
 
-See [installation](https://dj-stripe.dev/dj-stripe/2.7/installation/) instructions.
+See [installation](https://dj-stripe.dev/2.8/installation/) instructions.
```

Comment:
and i suggest instead of pointing to a specific version
we instead point to master
so instead of https://dj-stripe.dev/2.8/installation/
use
https://dj-stripe.dev/master/installation/